scania-digital-design-system/sdds
# feat(issue templates): New template for reporting bug, feature and questions

…ure or asking for help




**Describe pull-request**  
_Describe what the pull-request is about_

Added new github template based on the beta version with .yaml files instead of regular markdown. This will generate a form instead of a textbox.

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#1293](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1293)

**How to test**  
_Add description how to test if possible_
1. Go to .github folder in root or click [here](https://github.com/scania-digital-design-system/sdds/tree/improvement/new_issue_templates/.github/ISSUE_TEMPLATE)
2. Read the templates that have been added
3. See if something is miss or needs to be changed

**Additional context**  
_Add any other context about the pull-request here._
More templates can be added in the future
